README.md

```diff
@@ -132,6 +133,23 @@ for logger_name in ('snowflake.snowpark', 'snowflake.connector'):
     logger.addHandler(ch)
 ```
 
+## Reading and writing to pandas DataFrame
+
+Snowpark DataFrame API supports reading from and writing to a pandas DataFrame via the [to_pandas][to_pandas] and [write_pandas][write_pandas] commands. 
```

Comment:
I think it would be good for us to mention that both Snowpark Python API and Snowpark pandas API provides to_pandas() to convert Snowpark Dataframe and Snowpark DataFrame to native pandas Dataframe. However, if the  user is a pure Snowpark user, in other words, have an environment without modin option installed, users will need to install the [pandas] option to get interaction with pandas object work.

Reply:
we can also give a simple to_pandas example for snowpark pandas also, i believe we have example on our API doc

Reply:
Thanks! I've incorporated this in the updated README.